src/renderer/models/account.ts

```diff
@@ -0,0 +1,136 @@
+import {Keypair} from "@chainsafe/bls/lib/keypair";
+import {readdirSync, readFileSync} from "fs";
+import {ICGKeystore, ICGKeystoreFactory} from "../services/interfaces";
+
+export interface ISSZAccount {
+    name: string;
+    directory: string;
+    sendStats: boolean;
+}
+
+export class CGAccount implements ISSZAccount {
+    public name: string;
+    public directory: string;
+    public sendStats: boolean;
+
+    private unlocked = false;
+    private validators: Keypair[] = [];
+    private keystoreTarget: ICGKeystoreFactory;
+
+    constructor(name: string, directory: string, sendStats = false, keystoreTarget: ICGKeystoreFactory) {
+        this.name = name;
+        // Add / to the end if not provided
+        this.directory = directory + (directory.endsWith("/") ? "" : "/");
+        this.sendStats = sendStats;
+        this.keystoreTarget = keystoreTarget;
+    }
+
+    /**
+     * should return addresses from validator keystores in account directory
+     */
+    getValidatorsAddresses(): string[] {
+        // Loop trough files in account directory
+        const keystoreFiles: string[] = this.getKeystoreFiles();
+
+        const validatorAddresses: string[] = [];
+        keystoreFiles.forEach(file => {
+            const fileContents = readFileSync(this.directory + file);
+            const fileJSON = JSON.parse(fileContents.toString());
+            if (fileJSON.address) {
+                validatorAddresses.push(fileJSON.address);
+            }
+        });
```

Comment:
this will duplicate code. I would suggest something like:

```javascript
const keystorePaths = this.getKeystoreFiles();
keystorePaths.map(this.keystoreFactory.new).map(keystore => keystore,getAddress())
```

Reply:
make sure to handle parsing errors though